te_markdown/local-proxy-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Local Proxy Setup
+permalink: /te_markdown/local-proxy-setup/
+---
+
+Estimated reading time: 3 minutes
+
+
+## Setup environment variable
+The **Test-Editor-Web** can be used behind a proxy server. Therefore you need an environment variable named `http_proxy` which look like this:
```

Comment:
```suggestion
**Test-Editor-Web** can be used behind a proxy server. You'll need an environment variable named `http_proxy` which look like this:
```
(I think "therefore" means "for this reason" rather than "to this end")